_config.yml

```diff
@@ -46,6 +44,8 @@ navigation:
   url: /lean-product-design
 - text: Open Source Guide
   url: /open-source-guide
+ - text: "Partnership Playbook"
```

Comment:
Hey @wslack, do you mind removing the extra space in the beginning here just to keep formatting nice in the file?  I'll merge afterward. :-)  Thanks!